test/test_views.py

```diff
@@ -29,29 +29,58 @@ def setUp(self):
 	def test_rideGet(self):
 		response = self.test.get('/rides',headers = self.headers)
 		self.assertEqual(response.status_code,404)
+		self.assertTrue('There are no new rides' in response.data)
 
-	def test_rideGetNot(self):
+	def test_rideGet_data(self):
 		response = self.test.post('/rides',data=json.dumps(self.post_ride),headers = self.headers)
 		response = self.test.get('/rides',headers = self.headers)
 		self.assertEqual(response.status_code,200)
+		self.assertTrue('name' in response.data)
+		self.assertTrue('capacity' in response.data)
+		self.assertTrue('time' in response.data)
+		self.assertTrue('date' in response.data)
+		self.assertTrue('month' in response.data)
+		self.assertTrue('year' in response.data)
+		self.assertTrue('pick up point' in response.data)
+		self.assertTrue('destination' in response.data)
+		self.assertTrue('contact' in response.data)
 
 	def test_ridePost(self):
 		response = self.test.post('/rides',data=json.dumps(self.post_ride),headers = self.headers)
 		self.assertEqual(response.status_code,201)
+		self.assertTrue('Ride uploaded' in response.data)
 
 	def test_single_ride(self):
 		response = self.test.get('/rides/0',headers =self.headers)
 		self.assertEqual(response.status_code,200)
+		self.assertEqual(response.status_code,200)
+		self.assertTrue('name' in response.data)
+		self.assertTrue('capacity' in response.data)
+		self.assertTrue('time' in response.data)
+		self.assertTrue('date' in response.data)
+		self.assertTrue('month' in response.data)
+		self.assertTrue('year' in response.data)
+		self.assertTrue('pick up point' in response.data)
+		self.assertTrue('destination' in response.data)
+		self.assertTrue('contact' in response.data)
 
 	def test_single_rideNot(self):
 		response = self.test.get('/rides/-2',headers =self.headers)
 		self.assertEqual(response.status_code,404)
+		self.assertTrue('not found' in response.data)
 
 	def test_ride_request(self):
 		response =self.test.post('/rides/0/requests',data = json.dumps(self.post_request),headers = self.headers)
 		self.assertEqual(response.status_code,201)
+		self.assertTrue('name',response.data)
+		self.assertTrue('pickup time',response.data)
+		self.assertTrue('destination',response.data)
+		self.assertTrue('pick up point',response.data)
```

Comment:
I'm not sure if these *assertTrue* works
`assertTrue(expr, msg=None)`
I believe your "expressions" will always return True

Reply:
I also plan to add a sad test when the keys do not exist